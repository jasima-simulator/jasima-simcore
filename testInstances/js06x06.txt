# These data correspond to the instance ft06 (mt06) from:
#
#   H. Fisher, G.L. Thompson (1963), Probabilistic learning combinations
#   of local job-shop scheduling rules, J.F. Muth, G.L. Thompson (eds.),
#   Industrial Scheduling, Prentice Hall, Englewood Cliffs, New Jersey,
#   225-251 */
#
#   The optimal solution is 54

#number of machines
6

#number of routes
6

# machine orders <machine number(s)>;<setup state> (no space after or before comma/semicolon)
 3  1  2  4  6  5
 2  3  5  6  1  4
 3  4  6  1  2  5
 2  1  3  4  5  6
 3  2  5  6  1  4
 2  4  6  1  5  3

# processing times for each operation on a route
 1  3  6  7  3  6
 8  5 10 10 10  4
 5  4  8  9  1  7
 5  5  5  3  8  9
 5  3  5  4  3  1
 3  3  9 10  4  1 


# job specifications, jobs have to be ordered by release date
jobs

#number of jobs
6

# <route number> <release date> <due date> <weight>
1	0	0	1
2	0	0	1
3	0	0	1
4	0	0	1
5	0	0	1
6	0	0	1
