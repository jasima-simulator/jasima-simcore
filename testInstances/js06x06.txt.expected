testInstances/js06x06.txt IGF[SPT]	simtime	83.0
testInstances/js06x06.txt IGF[SPT]	cMax	83.0
testInstances/js06x06.txt IGF[SPT]	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[SPT]	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[SPT]	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[FCFS]	simtime	56.0
testInstances/js06x06.txt IGF[FCFS]	cMax	56.0
testInstances/js06x06.txt IGF[FCFS]	flowtimes	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[FCFS]	tardiness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[FCFS]	lateness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[PTPlusWINQPlusNPT]	simtime	83.0
testInstances/js06x06.txt IGF[PTPlusWINQPlusNPT]	cMax	83.0
testInstances/js06x06.txt IGF[PTPlusWINQPlusNPT]	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[PTPlusWINQPlusNPT]	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[PTPlusWINQPlusNPT]	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[WINQ]	simtime	77.0
testInstances/js06x06.txt IGF[WINQ]	cMax	77.0
testInstances/js06x06.txt IGF[WINQ]	flowtimes	49.83	36.00	77.00	15.48	299.00	6
testInstances/js06x06.txt IGF[WINQ]	tardiness	49.83	36.00	77.00	15.48	299.00	6
testInstances/js06x06.txt IGF[WINQ]	lateness	49.83	36.00	77.00	15.48	299.00	6
testInstances/js06x06.txt IGF[INV_FCFS]	simtime	61.0
testInstances/js06x06.txt IGF[INV_FCFS]	cMax	61.0
testInstances/js06x06.txt IGF[INV_FCFS]	flowtimes	50.50	32.00	61.00	12.53	303.00	6
testInstances/js06x06.txt IGF[INV_FCFS]	tardiness	50.50	32.00	61.00	12.53	303.00	6
testInstances/js06x06.txt IGF[INV_FCFS]	lateness	50.50	32.00	61.00	12.53	303.00	6
testInstances/js06x06.txt IGF[INV_SPT]	simtime	60.0
testInstances/js06x06.txt IGF[INV_SPT]	cMax	60.0
testInstances/js06x06.txt IGF[INV_SPT]	flowtimes	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[INV_SPT]	tardiness	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[INV_SPT]	lateness	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[WSPT]	simtime	83.0
testInstances/js06x06.txt IGF[WSPT]	cMax	83.0
testInstances/js06x06.txt IGF[WSPT]	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[WSPT]	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[WSPT]	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[INV_WSPT]	simtime	60.0
testInstances/js06x06.txt IGF[INV_WSPT]	cMax	60.0
testInstances/js06x06.txt IGF[INV_WSPT]	flowtimes	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[INV_WSPT]	tardiness	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[INV_WSPT]	lateness	51.83	45.00	60.00	5.56	311.00	6
testInstances/js06x06.txt IGF[EDD]	simtime	56.0
testInstances/js06x06.txt IGF[EDD]	cMax	56.0
testInstances/js06x06.txt IGF[EDD]	flowtimes	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[EDD]	tardiness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[EDD]	lateness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[FASFS]	simtime	56.0
testInstances/js06x06.txt IGF[FASFS]	cMax	56.0
testInstances/js06x06.txt IGF[FASFS]	flowtimes	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[FASFS]	tardiness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[FASFS]	lateness	50.83	38.00	56.00	6.55	305.00	6
testInstances/js06x06.txt IGF[SLK]	simtime	57.0
testInstances/js06x06.txt IGF[SLK]	cMax	57.0
testInstances/js06x06.txt IGF[SLK]	flowtimes	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[SLK]	tardiness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[SLK]	lateness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR]	simtime	57.0
testInstances/js06x06.txt IGF[INV_CR]	cMax	57.0
testInstances/js06x06.txt IGF[INV_CR]	flowtimes	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR]	tardiness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR]	lateness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR_V1]	simtime	84.0
testInstances/js06x06.txt IGF[INV_CR_V1]	cMax	84.0
testInstances/js06x06.txt IGF[INV_CR_V1]	flowtimes	48.67	31.00	84.00	18.52	292.00	6
testInstances/js06x06.txt IGF[INV_CR_V1]	tardiness	48.67	31.00	84.00	18.52	292.00	6
testInstances/js06x06.txt IGF[INV_CR_V1]	lateness	48.67	31.00	84.00	18.52	292.00	6
testInstances/js06x06.txt IGF[INV_CR_V2]	simtime	57.0
testInstances/js06x06.txt IGF[INV_CR_V2]	cMax	57.0
testInstances/js06x06.txt IGF[INV_CR_V2]	flowtimes	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR_V2]	tardiness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[INV_CR_V2]	lateness	51.17	38.00	57.00	6.74	307.00	6
testInstances/js06x06.txt IGF[ATC(k=1.0)]	simtime	83.0
testInstances/js06x06.txt IGF[ATC(k=1.0)]	cMax	83.0
testInstances/js06x06.txt IGF[ATC(k=1.0)]	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[ATC(k=1.0)]	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[ATC(k=1.0)]	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt Bremen1	simtime	76.0
testInstances/js06x06.txt Bremen1	cMax	76.0
testInstances/js06x06.txt Bremen1	flowtimes	48.83	37.00	76.00	15.54	293.00	6
testInstances/js06x06.txt Bremen1	tardiness	48.83	37.00	76.00	15.54	293.00	6
testInstances/js06x06.txt Bremen1	lateness	48.83	37.00	76.00	15.54	293.00	6
testInstances/js06x06.txt Bremen2	simtime	83.0
testInstances/js06x06.txt Bremen2	cMax	83.0
testInstances/js06x06.txt Bremen2	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt Bremen2	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt Bremen2	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[IFTMinusUITPlusNPT]	simtime	83.0
testInstances/js06x06.txt IGF[IFTMinusUITPlusNPT]	cMax	83.0
testInstances/js06x06.txt IGF[IFTMinusUITPlusNPT]	flowtimes	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[IFTMinusUITPlusNPT]	tardiness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt IGF[IFTMinusUITPlusNPT]	lateness	48.33	37.00	83.00	17.50	290.00	6
testInstances/js06x06.txt XWINQ	simtime	79.0
testInstances/js06x06.txt XWINQ	cMax	79.0
testInstances/js06x06.txt XWINQ	flowtimes	56.83	39.00	79.00	18.35	341.00	6
testInstances/js06x06.txt XWINQ	tardiness	56.83	39.00	79.00	18.35	341.00	6
testInstances/js06x06.txt XWINQ	lateness	56.83	39.00	79.00	18.35	341.00	6
